drones/image_processing/utils.py

```diff
@@ -34,7 +41,11 @@ def detect_object(image: np.ndarray) -> int:
     hsv_image = cv.cvtColor(image, cv.COLOR_BGR2HSV)
 
     # Mask in given color range.
-    mask = cv.inRange(hsv_image, LOWER_BOUND_COLOR, UPPER_BOUND_COLOR)
+    mask = cv.inRange(
+        hsv_image,
+        np.array([config.getint("LOWER_HUE"), config.getint("LOWER_SATURATION"), config.getint("LOWER_VALUE")]),
+        np.array([config.getint("UPPER_HUE"), config.getint("UPPER_SATURATION"), config.getint("UPPER_VALUE")]),
+    )
```

Comment:
i think that putting array on config will make this code clearer, but someone else may look at it

Reply:
As I was searching about putting array into config I didn't find an easy way to do that. For me the way as it is implemented now is ok, but it was my first experience with .ini config file, so I might be wrong :)

Reply:
Ok, so if we want array in config it would something like that:

np.asarray(config['LOWER_BOUND'].split(' '), dtype=np.int),
np.asarray(config['UPPER_BOUND'].split(' '), dtype=np.int)

and in ini file it would be:

LOWER_BOUND = 0 220 100
UPPER_BOUND = 15 255 255

is this seems clearer?

Reply:
for me yes